### DISCUSSION TITLE
Streams "bounce" online/offline

### DISCUSSION THREAD

**Discussion by naanlizard** [OvenMediaLabs/OvenMediaEngine#1735] [Help]
We've been trying to track down the source of a bug for a while and aren't having much luck

Essentially, it looks like OBS disconnects/reconnects nearly instantly for several minutes, and eventually calms down.

Here are some logs from OME

[Explore-logs-2024-11-04 20_16_27.txt](https://github.com/user-attachments/files/17625640/Explore-logs-2024-11-04.20_16_27.txt)

Any thoughts on how to debug this? We haven't found a way to reproduce it, it's just random when it happens

**Comment by bchah** [comment]
Your setup is managing an impressive number of connections. I assume you've already checked and aren't hitting socket / open file / other low-level resource limits? Are you able to duplicate your configuration in a staging environment and run two instances of OBS against prod/staging to see if you can reproduce the disconnect yourself?
